package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "powerbi-visuals-bulletchart",
   "description": "Bullet chart",
-  "version": "1.4.5",
+  "version": "1.4.6",
   "author": {
     "name": "Microsoft",
     "email": "pbicvsupport@microsoft.com"
   },
   "scripts": {
-    "postinstall": "typings install && pbiviz update 1.3.0",
+    "postinstall": "typings install && pbiviz update 1.6.0",
```

Comment:
If you want to get rid of typings module. You have to remove it in package.json and use npm TS modules instead.